logstash-core/src/main/java/org/logstash/secret/store/backend/JavaKeyStore.java

```diff
@@ -370,11 +372,14 @@ private void releaseLock(Lock lock) {
      * Saves the keystore with some extra meta data if needed. Note - need two output streams here to allow checking the with the append flag, and the other without an append.
      */
     private void saveKeyStore() throws IOException, CertificateException, NoSuchAlgorithmException, KeyStoreException {
-        FileLock fileLock;
+        FileLock fileLock = null;
```

Comment:
@jakelandis I think you can define this inside the first `try` for better readability right?

Reply:
done.